public/js/controllers/StatsController.js

```diff
@@ -414,17 +424,27 @@ angular.module('BlocksApp').controller('StatsController', function($stateParams,
                 .attr("x", 9)
                 .attr("dy", ".35em");
 
+            svg.call(tip);
 
             svg.append("rect")
                 .attr("class", "overlay")
                 .attr("width", width)
                 .attr("height", height)
-                .on("mouseover", function() { focus.style("display", null); })
-                .on("mouseout", function() { focus.style("display", "none"); })
+                .on("mouseover", function(d) {
+                    var x0 = x.invert(d3.mouse(this)[0]);
+                    var s1 = _.minBy(data, function(d) {
+                        return Math.abs(moment(x0).unix()-d.unixtime);
+                    });
+                    tip.show(s1, this);
+                    tip.style("left", d3.event.pageX + 10 + "px");
+                    tip.style("top", d3.event.pageY + 25 + "px");
+                    focus.style("display", null);
+                 })
+                .on("mouseout", function() { tip.hide(); focus.style("display", "none"); })
                 .on("mousemove", mousemove);
 
 
-            function mousemove() {
+            function mousemove(d) {

```

Comment:
Same question as the one below. Why pass a `d` function parameter if it's not being used?